lib/item_repository.rb

```diff
@@ -29,8 +87,6 @@ def get_info(row)
     }
   end
 
-  private
```

Comment:
@aschwartz1 @jbranham1 are we wanting to remove private? In the merchant repo, it looks like this:
![Screen Shot 2021-01-06 at 4 43 22 PM](https://user-images.githubusercontent.com/48256547/103827303-4b82f480-503e-11eb-9e54-1cafa11b7f75.png)

Reply:
I didn't know we had an opinion on this yet!

I like the idea of privatizing methods to control the public interface of our objects.  However, I really don't know what the good Rubyist does... so I will make a point to ask about this in our project check-in.

Reply:
For now can we go ahead and check this in and put this in our list of questions? @abreaux26 @aschwartz1

Reply:
Agree